easybuild/easyconfigs/v/vawk/vawk-0.0.1.eb

```
@@ -0,0 +1,43 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'Tarball'
+
+name = 'vawk'
+version = '0.0.1'
+
+homepage = 'https://github.com/cc2qe/vawk'
+description = """An awk-like VCF parser 
+"""
+
+toolchain = SYSTEM  # or = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://github.com/cc2qe/vawk/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['90cf5a57428ccaf3490ac98fc5c9f5f2c11ea2d1824188d9310bd34a8ef18d6f']
+
+postinstallcmds = [
+    "chmod +x %(installdir)s/%(namelower)s",
+    "sed -i '1 i#!/usr/bin/env python' %(installdir)s/%(namelower)s",
+]
+
+# as this app does not have a bin subdirectory we just need to add the root dir to PATH
+modextrapaths = {'PATH': ''}
```

A couple of suggestions here:
* You can avoid making all files in the installation directory available through `PATH` by moving the `vawk.py` script into `bin`
* there is a specific parameter to fix the shebang of python scripts, no need to manually `sed` it
```suggestion
    "mkdir %(installdir)s/bin && mv %(installdir)s/%(name)s %(installdir)s/bin/",
]

fix_python_shebang_for = ['bin/%(name)s.py']
```